docs/gno-tooling/cli/gnokey.md

```diff
@@ -207,6 +207,20 @@ gnokey maketx call \
     > unsigned.tx
 ```
 
+:::info `call` is a state-changing message  
```

Comment:
This should be a `warn`, instead of an `info`

Reply:
[3ca739e](https://github.com/gnolang/gno/pull/2134/commits/3ca739ef0f7e85983e3a3a60673b8422e16006af)